packages/directory/addon/routes/directory/my-directory.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+import { get } from '@ember/object';
+import { A as arr } from '@ember/array';
+import { run } from '@ember/runloop';
+
+export default Route.extend({
+  /**
+   * @property { Service } user
+   */
+  user: inject(),
+
+  async _fetchItems(user, type){
+    let reports,
+        dashboards,
+        items = arr();
+
+    if(type === null || type === 'reports'){
+      reports = await get(user, 'reports');
+      run(() => items.push(...reports.toArray()));
+    }
+    if(type === null || type === 'dashboards'){
+      await run(async () => {
+        dashboards = await get(user, 'dashboards');
+        items.push(...dashboards.toArray())
+      });
+    }
+
+    return items.sortBy('title');
```

Comment:
Can sort me a query param? Also sort by `updatedOn ` by default?

Reply:
I will add the `sort` query param in the second phase. Since now we are sorting by `updatedOn` by default, do we need the option in the sidebar?